docs/crd-changelog.md

```diff
@@ -0,0 +1,160 @@
+# CRD Changelog
+
+This document explains major changes made in new CRD versions. It is intended to help users migrate and take
+advantage of the new features.
+
+## OpenTelemetryCollector.opentelemetry.io/v1beta1 
+
+### Migration
+
+There is no need for any immediate user action. The operator will automatically convert existing `v1alpha1` resources to `v1beta1`.
+
+However, the conversion procedure will leave your resources stored as both versions. Support for `v1alpha1` will be removed in the future.
```

Comment:
This is not accurate, but rather misleading. 

>  the conversion procedure will leave your resources stored as both versions.

The custom resources stored as v1alpha1 are not automatically updated to v1beta1. The updated or newly written cutom resources will be stored as v1beta1 because it is the new storage version. 

I would also link https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#upgrade-existing-objects-to-a-new-stored-version

Reply:
Ok, how about:

```
There is no need for any immediate user action. The operator will continue to support existing `v1alpha1` resources.

In addition, any newly applied `v1alpha1` resource will be converted to `v1beta1` and stored as both versions. The plan is to remove support for `v1alpha1` in a future version, so users should migrate promptly.

(detailed migration guide here)
```